### ISSUE TITLE
efs provisioner, nfs mount becomes unresponsive/super slow

### ISSUE THREAD

**Issue by jurgenweber** [kubernetes-incubator/external-storage#355]
```
Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.5", GitCommit:"17d7182a7ccbb167074be7a87f0a68bd00d58d97", GitTreeState:"clean", BuildDate:"2017-08-31T09:14:02Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.4", GitCommit:"793658f2d7ca7f064d2bdf606519f9fe1229c381", GitTreeState:"clean", BuildDate:"2017-08-17T08:30:51Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
```

```
image: quay.io/external_storage/efs-provisioner:v0.1.2
```

Using the Prometheus helm chart, I use the values: 
```
storageClass: "aws-efs"
```

This works but over time the share becomes slow and Prometheus starts to fail/become unresponsive. I expereinced this late last week and I found the share in the efs-provisioner container was responding fine but the mount in the prometheus server container was super slow but today.. both are super slow but inside the efs-provisioner container is faster than outside.

**Comment by jurgenweber** [comment]
I have a feeling I am getting hampered by:

using a t2 instance with low networking.
my not realising that EFS has a burst system; http://docs.aws.amazon.com/efs/latest/ug/performance.html#performancemodes

my FileSystem is only 10gb or so, so I get a hour of burst. More to come as I discover more but any feedback is welcomed.

**Comment by wongma7** [comment]
Yes, I think it is burst credits at work. While you've mounted the efs share, the provisioner is not doing anything, you are interacting directly with aws efs.

**Comment by imesh** [comment]
@jurgenweber Thanks for sharing your experience with this. I also found the same issue on AWS while trying to use Kubernetes EFS provisioner. May I know whether you were able to resolve this problem by changing the instance type to a one which has higher network bandwidth?

**Comment by jurgenweber** [comment]
Changing the node did not help, basically you need filesystems over a TB in size for EFS to be useful it is all about its burst system. I abandoned this approach for just using EBS/gp2 and fixing applications to an AZ.

**Comment by amoskyler** [comment]
I just ran into this problem I believe. I'm not using external-storage, but have simply created a PV and PVC, attaching my EFS as NFS.

The mount essentially stalls completely, to the point my influxdb instance errors out and quits.

I'm a bit confused what exactly might be causing this issue.
